src/config/frontend/EdFi.DmsConfigurationService.Frontend.AspNetCore/Model/TokenRequest.cs

```diff
@@ -9,15 +9,21 @@ namespace EdFi.DmsConfigurationService.Frontend.AspNetCore.Model;
 
 public class TokenRequest
 {
-    public string? ClientId { get; set; }
-    public string? ClientSecret { get; set; }
+    public string client_id { get; set; } = "";
```

Comment:
You are supposed to be able to put an attribute on these such as `[FromForm(Name = "client_id")]` , but it doesn't seem to work. I think it is a bug in the framework. So I had to name these properties to match the reqest.